x-pack/plugins/security_solution/public/common/components/event_details/cti_details/helpers.tsx

```diff
@@ -81,7 +81,7 @@ export const getEnrichmentIdentifiers = (enrichment: CtiEnrichment): CtiEnrichme
   field: getEnrichmentValue(enrichment, MATCHED_FIELD),
   value: getEnrichmentValue(enrichment, MATCHED_ATOMIC),
   type: getEnrichmentValue(enrichment, MATCHED_TYPE),
-  provider: getShimmedIndicatorValue(enrichment, PROVIDER),
+  provider: getShimmedIndicatorValue(enrichment, FEED_NAME),
```

Comment:
I think we should rename this to `feedName` to remain consistent. I see no reason not to keep returning `provider` as well:

```suggestion
  feedName: getShimmedIndicatorValue(enrichment, FEED_NAME),
  provider: getShimmedIndicatorValue(enrichment, PROVIDER),
```